### PR TITLE
add open sgid db connection page

### PR DESCRIPTION
**Pull request by gregbunce**

### CODE REVIEW

**Comment by gregbunce** [comment]
i'm going to left justify the headings on the page, but let me know what y'all think.  also, @steveoh I'll leave the website structure to you as to where you want the page to live.  i created a new dir, but feel free to move it where best.  (@eneemann @ZachBeck @stdavis )

**Comment by gregbunce** [comment]
i suppose we could even change the page URL to //connect-to-open-sgid and place it in sgid-database dir. ?

File: open-sgid-database/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+### Before you Connect...
+{: .text-left }
+
+- Please take a moment to read over the [Open SGID Terms of Use]( {% link about/policy/open-sgid/index.md %}).
+- Get familiar with the [Open SGID GitHub repository](https://github.com/agrc/open-sgid) where you can troubleshoot and track known issues.
+
+### How to Connect
+{: .text-left }
+
+There are a variety of clients and application that connect to a PostgreSQL/PostGIS database. Here's how to connect using a few of the more common applications (ArcGIS Desktop, ArcGIS Pro, DBeaver, and QGIS).
```
Comment (nathankota) [inline open-sgid-database/index.md:28]: Add _"s"_ to pluralize _applications_ in the first sentence

Comment (steveoh) [reply open-sgid-database/index.md:28]: tip: use the suggestion feature for little modifications like this.

**Comment by steveoh** [comment]
thanks, i'll take a look this afternoon

File: open-sgid-database/index.md (hunk 1 above)
Comment (nathankota) [inline open-sgid-database/index.md:28]:
```suggestion
There are a variety of clients and applications that connect to a PostgreSQL/PostGIS database. Here's how to connect using a few of the more common applications (ArcGIS Desktop, ArcGIS Pro, DBeaver, and QGIS). 
```

**Comment by eneemann** [comment]
Should we remove the bullets from the images?  I think that would look a little cleaner.  Also is DBeaver that popular?  I was thinking maybe QGIS should go before DBeaver, since we're GIS folk and all.

**Comment by steveoh** [comment]
> Should we remove the bullets from the images? I think that would look a little cleaner. Also is DBeaver that popular? I was thinking maybe QGIS should go before DBeaver, since we're GIS folk and all.

It appears to be done in alphabetical order to hopefully remove the idea of popularity ratings.

**Comment by gregbunce** [comment]
yup, i went alphabetical, but it is kind of silly that it's an open source db and we have arc stuff at the top.  we could either reverse the order or put the mapping applications first (desktop, pro, qgis) and then the db tool (dbeaver).  thoughts?

**Comment by steveoh** [comment]
I can't find the thread but i'm going to adjust the structure some. I've moved everything inside of sgid like we talked about somewhere.

**Comment by steveoh** [comment]
Ok, when the preview finishes, go to https://deploy-preview-1402--gis-utah-gov.netlify.app/sgid. 

I want to have a top level gis.utah.gov/sgid url. I think it's nice and we do it for most of our other gold star services.

The /sgid landing page is whatever. @atribolet feel free to have creative control over what that says. I copy and pasted some text there but mainly wanted it as a place we could link out to all the variations of the SGID. 

I'm not convinced on the name of the state network sgid10 and would be open to suggestions for a title for that and a slug for the url. 

Thanks for the review folks @eneemann @ZachBeck @stdavis @nathankota

**Comment by gregbunce** [comment]
nice, i like everything on one page.  where would that 'sgid' page be exposed (how would i discover or navigate to it)?

**Comment by steveoh** [comment]
> nice, i like everything on one page. where would that 'sgid' page be exposed (how would i discover or navigate to it)?

I think a link on https://gis.utah.gov/#data and https://gis.utah.gov/data would be a good starting point. Do you have some ideas?

**Comment by gregbunce** [comment]
yup, that would be my vote as well.

**Comment by gregbunce** [comment]
i like what i see.  and thanks for removing the bullets and adding the numbered lists.  it looks good to me (i'm happy with it).  any discussion on the application order (@ZachBeck @nathankota @eneemann @stdavis )... ie: should we rework the current order of desktop, pro, dbeaver, and qgis?  also, let me know when you want me to change it from draft pr to a pr.

**Comment by steveoh** [comment]
> any discussion on the application order (@ZachBeck @nathankota @eneemann @stdavis )... ie: should we rework the current order of desktop, pro, dbeaver, and qgis? 

I think alphabetical is best. Maybe you can say here are some applications in alphabetical order to solve any ranking issues.

> also, let me know when you want me to change it from draft pr to a pr.

I think we are still in the draft phase so I'd leave it.

**Comment by nathankota** [comment]
your justification for open source first, followed by esri products was legitimate to me but, ultimately, I'm indifferent

**Comment by ZachBeck** [comment]
My vote would be for what the end user would most likely be connecting with: ESRI products, QGIS, then DBeaver. But in the end I'm not too worried about the order.

**Comment by steveoh** [comment]
Am i driving the last changes home or are you greg?

**Comment by gregbunce** [comment]
i'm happy with it.  but, i guess the remaining discussion on the table is the order of the application screenshots.  i think that's it right?  from my perspective, i say we keep it simple and we leave it alphabetical.  i'm open to a change, though, if someone really wants a different order.

**Comment by steveoh** [comment]
I think then the links to the page. do we want to do that in a separate pr?

**Comment by gregbunce** [comment]
right... good point.  yeah, maybe let's do a separate pr.  is that alright?  that way we/I can digest it and see where they might best fit.  in this pr, there's a link to the page on the data page in the Open SGID section (Get Connected). but, yeah, we should add links to the page on the TOS, some blog posts, etc.. ?

**Comment by gregbunce** [comment]
thanks.  also, i'm happy to create the branch and get a new pr going.

**Pull request merged**